### PR TITLE
Correct link to former react.rocks website

### PR DESCRIPTION
**Pull request by webwelten**
react.rocks has moved to js.coach and the packages at react.rocks aren't up to date anymore.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ Note that you'll need the Android NDK installed, see [prerequisites](https://git
 
 ## Extending React Native
 
-- Looking for a component? [react.parts](http://react.parts/)
+- Looking for a component? [JS.coach](https://js.coach)
```
Comment (corbt) [inline README.md]: I think it would be better to change the URL to `https://js.coach/react-native`

**Comment by webwelten** [comment]
You're right. react.parts redirects to /native as well

**Comment by corbt** [comment]
Cool, and welcome to the React Native project :smile:. Just two more steps if you don't mind, and we'll get this committed:

1. Sign the Facebook [CLA](https://code.facebook.com/cla)
2. Squash your two commits into a single one.

**Comment by corbt** [comment]
And for anyone wondering if this move is legit, here's the current header from [react.parts](https://react.parts/native):

screen shot 2016-01-20 at 5 14 49 pm

**Comment by mkonicek** [comment]
Cool, feel free to merge :)

**Comment by mkonicek** [comment]
Thanks @webwelten and @corbt!

**Pull request merged**